data/human/culture conversations.txt

```diff
@@ -805,7 +804,7 @@ mission "Militia returning to Solace"
 mission "Republic 300 Year Anniversary"
```

Comment:
Another unrelated change I spotted: `(Go on about your day as usual.)` -> `(Go on about my day as usual.)`, since action-choices should be in first person. (The order of the labels in this conversation is also very odd.)